docs/tutorials/rootless_tutorial.md

```diff
@@ -106,8 +106,32 @@ Once the Administrator has completed the setup on the machine and then the confi
 
 ### User Configuration Files
 
-The Podman configuration files for root reside in `/usr/share/containers` with overrides in `/etc/containers`.  In the rootless environment they reside in `${XDG_CONFIG_HOME}/containers` (usually `~/.config/containers`) and are owned by each individual user.  The main files are `libpod.conf` and `storage.conf` and the user can modify these files as they wish.
+The three main configuration files are `containers.conf`, `storage.conf` and `registries.conf`.
 
+#### containers.conf
+Podman reads
+1. `/usr/share/containers/containers.conf`
+2. `/etc/containers/containers.conf`
+3. `$HOME/.config/containers/containers.conf`
+
+if they exist in that order. Each file can override the previous for particular fields.
+
+#### storage.conf
+For `storage.conf` the order is
+1. `/etc/containers/storage.conf`
+2. `$HOME/.config/containers/storage.conf`
+
+In rootless podman certain fields in `/etc/containers/storage.conf` are ignored.
```

Comment:
We should list the fields that are ignored.
```
       graphroot=""
         container storage graph dir (default: "/var/lib/containers/storage")
         Default directory to store all writable content created by container storage programs.

       runroot=""
         container storage run dir (default: "/var/run/containers/storage")
         Default directory to store all temporary writable content created by container storage programs.
```

Reply:
I suggest adding their default values, when in rootless podman. (see new commits). I do not exactly know, if podman really reads out $XDG_RUNTIME_DIR or just defaults to `/run/user/$UID`.

Reply:
It uses XDG_RUNTIME_DIR